### ISSUE TITLE
Add unique token to jobfile.xml for non logged in access

### ISSUE THREAD

**Issue by a-tze** [crs-tools/tracker#95]
*Issue migrated from trac ticket # 95*

**component:** unknown | **priority:** minor

**Comment by a-tze** [comment]
@jjeising ping :)

**Comment by jjeising** [comment]
@pegro Yes, this is issue is still valid. The jobfile linked on the encoding ticket page here
<img width="859" alt="ep_dl_job" src="https://user-images.githubusercontent.com/40333/32698527-a004f0d0-c7a7-11e7-87bc-c78eadf1c512.png">
is currently bound to the user login. Because of the nature of the jobfile it may be helpful to download the file on the commandline.

Therefore it would be helpful to exclude the jobfile output from the login requirement and add a per ticket unique token for authentication.

**Comment by pegro** [comment]
I'm not sure I get the use case of this.

**Comment by a-tze** [comment]
user is in browser, looks at encoding ticket. user copies link of "download jobfile" into clipboard, wants to do a "wget $url" on an encoding machine via SSH to execute it manually ... wget gets "forbidden". current workaround is to copy and paste the whole XML content.

**Comment by pegro** [comment]
Ah, makes sense.
